src/Notes.pro

```diff
@@ -41,16 +41,15 @@ HEADERS  += \
 FORMS += $$PWD/mainwindow.ui
 RESOURCES += $$PWD/images.qrc
 
-unix:!macx {
-    isEmpty (PREFIX) {
-        PREFIX = /usr
-    }
-
-    BINDIR = $$PREFIX/bin
-    DATADIR =$$PREFIX/share
-
-    target.path = $$BINDIR
-    INSTALLS += target
+linux:!android {
```

Comment:
Not good, you should give the user the choice to install the app wherever he/she wants.